### PR TITLE
Fix Type Issue In Ethtool Test

### PR DESCRIPTION
**Pull request by sethAmazon**
# Description of the issue
Ethtool test never ran.

# Description of changes
Run ethtool test. Fix type issue in test. 

# Tests
Ran unit test in intelij. 

# Requirements
_Before commit the code, please do the following steps._
1. Run `make fmt` and `make fmt-sh`
2. Run `make linter`

### CODE REVIEW

File: translator/translate/metrics/metrics_collect/ethtool/ethtool_test.go [hunk 1]
```diff
@@ -24,33 +24,50 @@ func TestDefaultConfig(t *testing.T) {
...
-	e := json.Unmarshal([]byte(`{"ethtool": {
+	err := json.Unmarshal([]byte(`{"ethtool": {
 					"interface_include": [
 						"eth0"
 					],
 					"interface_exclude": [
 						"eth1"
 					],
 					"metrics_include": [
-						"bw_in_allowance_exceeded",
```
Comment (sethAmazon) [inline translator/translate/metrics/metrics_collect/ethtool/ethtool_test.go:41]: This is invalid json.

**Review by ZhenyuTan-amz — APPROVED** [review]
Changing e to err makes it much more clear.

File: translator/translate/metrics/metrics_collect/ethtool/ethtool_test.go [hunk 2]
```diff
@@ -13,9 +13,9 @@ import (
 func TestDefaultConfig(t *testing.T) {
 	d := new(Ethtool)
 	var input interface{}
-	e := json.Unmarshal([]byte(`{"ethtool": {
+	err := json.Unmarshal([]byte(`{"ethtool": {
 					}}`), &input)
-	if e == nil {
+	if err == nil {
```
Comment (khanhntd) [inline translator/translate/metrics/metrics_collect/ethtool/ethtool_test.go:18]:
```suggestion
	assert.NoError(t, err)
```

File: translator/translate/metrics/metrics_collect/ethtool/ethtool_test.go [hunk 3]
```diff
@@ -24,33 +24,50 @@ func TestDefaultConfig(t *testing.T) {
...
 						"eth1"
 					],
 					"metrics_include": [
-						"bw_in_allowance_exceeded",
-					],
+						"bw_in_allowance_exceeded"
+					]
 					}}`), &input)
-	if e == nil {
+	if err == nil {
```
Comment (khanhntd) [inline translator/translate/metrics/metrics_collect/ethtool/ethtool_test.go:46]:
```suggestion
	assert.NoError(t, err)
```

**Pull request merged**